### PR TITLE
1357 - Clarification regarding token transfers

### PR DESCRIPTION
**Pull request by ipopescu**
### What does this PR fix/introduce?

As requested in [this thread](https://casper-labs-team.slack.com/archives/C01J86T699U/p1705923063189069?thread_ts=1705837279.975239&cid=C01J86T699U), I added a note clarifying that token transfers must be handled via stored contract (not stored session).

### Checklist

- [x] Docs are successfully building - `yarn install && yarn run build`.

### Reviewers
@ACStoneCL

### CODE REVIEW

File: source/docs/casper/resources/advanced/transfer-token-to-contract.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ slug: /resources/tutorials/advanced/transfer-token-to-contract
 
 This tutorial covers two methods to handle tokens via a contract. This is not a native process to a Casper network and will require the use of custom code. The following two scenarios provide a framework for developers and the pros and cons of each example. Developers should choose the option that best suits their individual needs.
 
+For increased security, token transfers must be handled via contract code, not session code. As a result, `transfer-*` methods are unavailable in stored session code, even when the stored session runs in the account context.
```
Comment (davidatwhiletrue) [inline source/docs/casper/resources/advanced/transfer-token-to-contract.md:10]: @ipopescu the restriction applies only to transfers from the main purse in the account, not all transfers. On the other hand, it's the opposite way. "token transfers must be handled via session code, not contract code".

**Review by davidatwhiletrue — CHANGES_REQUESTED** [review]

File: source/docs/casper/resources/advanced/transfer-token-to-contract.md (hunk 1 above)
Comment (ipopescu) [reply source/docs/casper/resources/advanced/transfer-token-to-contract.md]: Added this clarification with commit [1e77910](https://github.com/casper-network/docs/pull/1356/commits/1e77910ff00f506fe1449b0bcd9e32658ffede02). Please take a look, @davidatwhiletrue.

File: source/docs/casper/resources/advanced/transfer-token-to-contract.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ slug: /resources/tutorials/advanced/transfer-token-to-contract
 
 This tutorial covers two methods to handle tokens via a contract. This is not a native process to a Casper network and will require the use of custom code. The following two scenarios provide a framework for developers and the pros and cons of each example. Developers should choose the option that best suits their individual needs.
 
+For increased security, token transfers from a main purse must be handled via contract code, not session code as in [other cases](../../developers/writing-onchain-code/writing-session-code.md#example-3-transfers-using-session-code-transfers-using-session-code). In this case, `transfer-*` methods are unavailable in stored session code, even when the stored session runs in the account context.
```
Comment (davidatwhiletrue) [inline source/docs/casper/resources/advanced/transfer-token-to-contract.md:10]: @ipopescu  "handled via session code, not contract code", right?

For increased security, token transfers from an account main purse must be handled via **session code** as in [other cases](../../developers/writing-onchain-code/writing-session-code.md#example-3-transfers-using-session-code-transfers-using-session-code). Therefore, `transfer-*` methods are unavailable for tokens originated from an account main purse in stored session code, even when the stored session runs in the account context.

Comment (Jiuhong-casperlabs) [reply source/docs/casper/resources/advanced/transfer-token-to-contract.md:10]: I would change a little
> token transfers from a main purse must be handled via contract code
=>

token transfers from a main purse must be handled via WASM code

Comment (ipopescu) [reply source/docs/casper/resources/advanced/transfer-token-to-contract.md]: I've changed the wording with commit 76bd4e3. We don't have stored sessions anymore. What do you think?

Comment (davidatwhiletrue) [reply source/docs/casper/resources/advanced/transfer-token-to-contract.md]: Looks good to me. Thanks @ipopescu

**Review by davidatwhiletrue — APPROVED** [review]

**Comment by ipopescu** [comment]
I will merge this to close off the week. If there is any further feedback, I can always open another PR.

**Pull request merged**